src/containers/WaGroups/WaManagedPhones/WaManagedPhones.module.css

```
@@ -24,18 +25,15 @@
-.syncButton {
-  margin: 0 8px;
-  cursor: pointer;
-  height: 36px;
-  border: 1px solid #cccccc;
-  border-radius: 27px;
-  background: #ffffff;
-  box-shadow: none;
-  color: #2ea36a;
-  font-size: 14px;
-  width: 25%;
+.Buttons Button {
```

Stylelint (17.14.0)

[error] Expected "Button" to be "button" (selector-type-case)

(selector-type-case)